fl4health/privacy_mechanisms/docs/index.md

```diff
@@ -0,0 +1,89 @@
+# Privacy Mechanisms
+
+!!! info ""
+
+    A **privacy mechanism** is a randomized algorithm which takes as input a user's data, and through injecting noise to the data, the algorithm produces a privitized version of the users data as ouput, which can subsequently be shared in the clear for tasks such as federated learning.
+
+The following discrete privacy mechanisms have been implemented to privitize clients' discrete vectors for the cryptographic primitive called secure aggregation (SecAgg) and its variations.
+
+1. Discrete Gaussian mechanism
+2. Poisson binomial mechanism
+3. Skellam mechanism
+4. Binomial mechanism
+
+## 1. Discrete Gaussian Mechanism
+
+This is an additive mechanism that works on discrete user data $x$ defined by $\mapsto x + Z$ where $Z\sim \mathcal{N}_{\mathbb{Z}}(\mu, \sigma ^2)$ is a random variable that takes values in $\mathbb{Z}$ with the following probability mass function
+
+$$\mathbb{P}[Z = n] = \frac{e^{ - \frac{(n-\mu)^2}{2\sigma ^2}}}{\sum_{m\in\mathbb{Z}} e^{ - \frac{(m-\mu)^2}{2\sigma ^ 2}}} \  $$
+
+for each $n \in \mathbb{Z}$. We used **rejection sampling** for drawing samples from the discrete Gaussian distribution. The mechanism is exposed through the following function
+
+``` py title="dpDiscreteGaussian.py" linenums="1"
+def DiscreteGaussianMechanism(query_vector: List[int],
+                                variance: float) -> List[int]:
+
+```
+
+## 2. Poisson Binomial Mechanism
+
+
+!!! info inline end "Note"
```

Comment:
I'm not very familiar with this annotation for markdown, but it seems like maybe this is orphaned or in an odd spot, given where `!!! info ""` starts. I might be wrong though?